pkgs/development/cuda-modules/lib/default.nix

```
@@ -0,0 +1,11 @@
+{ lib }:
```

Let's make it static?

```suggestion
let
  lib ? import ../../../../lib;
in
```

---

Good suggestion -- implemented in latest.